### DISCUSSION TITLE
Classe Paladin

### DISCUSSION THREAD

**Discussion by LaMangousteRose** [Crownicles/Crownicles#2363] [Nouvelles idées]
Je propose de changer le nombre de dégats des attaques béliers car en plus de l'étourdissement ( parfois ), on ne peux pas attaquer donc on se prends environ 300 dégats et l'attaquant ( le paladin ) s'en inflige 80.
En plus de cela : quand le paladin se sent en danger il fait une attaque ultime où il sait qu'au prochain tour il va mettre beaucoup de dégats.
C'est pour ces raisons que je propose de changer le nombre de dégats des attaques de la classe Paladin.

**Comment by JustinPelpoir** [comment]
Bonsoir,
En tant que Paladin je ne peux que réfuter ces propos.

• Une attaque bélier ne tape jamais du 300, la preuve quand moi lvl70 avec deux objets légendaires atk/défense je tape mon apprenti lvl20 en amical il prend max 230 par bélier. Pour les combats classés que je fais ça tourne entre 130 et 180.
•  L'étourdissement est aléatoire, le paladin peut très bien s'étourdir soi-même tout en tapant du 30 et en prenant 80 de contrecoup. Ça fait qu'un Paladin peut tout aussi vaincre plus fort que soit que perdre contre plus faible.
• Le Paladin n'a pas besoin d'un nerf, un léger buff de la divine peut-être, le reste c'est bien comme ça.
• En tant que paladin lvl70, avec pour mon niveau des stats plus que respectable grâce à ma luck sur les items :), je peux témoigner que Paladin est loin d'être broken, paladin ne gagne pas contre mages et plusieurs classes sont très efficaces pour le contrer

la classe a déjà subit un gros nerf avec le bye bye de l'attaque bouclier, je passe de spam 210 dégâts fixe avec -70% atk en face, à 200 en face, -70 chez moi (avec un taux de fail bien plus élevé), et un stun une fois sur deux

j'en profite pour demander un changement de la Divine, un léger buff pls :)

**Reply by Procion06** [reply]
en tant que mage je ne peux que réfuter ces propos xD

le paladin gagne contre les mages
"plusieurs classes sont très efficaces pour le contrer" Lesquelles ?

"la classe a déjà subit un gros nerf avec le bye bye de l'attaque bouclier"
> la classe a bénéficié d'un gros buff qui lui permet de sortir une attaque à +800 de dégâts !
> la classe a bénéficié d'un gros buff qui lui permet de faire 5 stuns ou plus durant un combat 

Défendre sa classe c'est bien mais faut garder un minimum d'objectivité ^^

**Comment by LaMangousteRose** [comment]
Quand j'ai dit 300 dégats, c'était sur 2 tours.
Et surtout chaque fois que j'ai failli vaincre ou vu quelqu'un presque gagner contre un paladin, le paladin utilise une attaque ultime à la dernière minute pour etre sur de gagner le combat.

**Reply by ghost** [reply]
Bonsoir bonsoir,

En tant que Paladin je peux dire que l'attaque bélier est très bien comme elle est actuellement, même que son taux de précision devrait être selon moi un peu plus haut. En ce qui concerne les fois où tu es stun c'est plutôt rare donc ça va. Et oui, une attaque ultime fait très mal car c'est une attaque en deux tours. De plus le paladin a un désavantage lors des combats, il ne peut pas du tout se soigner contrairement aux autres classes (sauf mage) et le mage lui compense ça avec des attaques en plusieurs tours (feu, poison, maudite) donc heureusement que le paladin a des attaques tels que la divine ou l'ultime.

**Comment by Gabrielre14** [comment]
Bon. A mon tour, en tant que seul paladin lvl 100+ du top gloire, de donner mon avis... 

1) Points forts / points faibles du paladin. 
Le paladin est une classe puissante en PvP. Elle claque, je ne peux le nier. Elle possède de quoi counter certaines attaques dévastatrices, tel que le combo canon avec la bélier et les attaques à chargement avec la bouclier. Le paladin a aussi un burst incroyable en fin de fight avec le combo ultime + double divine. Le problème étant qu'il ne peut en faire qu'un des deux. En effet, si on utilise trop de bélier, ou 1 ou 2 bouclier, on a plus assez de souffle en fin de fight pour finaliser le combo ultime + divine. Et sans ce combo, il ne peut rien faire. L'attaque ultime est une attaque sur 2 tours à 10 de souffle, qui ne peut être lancé que dans des conditions précises. Reporté aux dégats / seconde, sans crit les dégats sont largement inférieurs à l'attaque sombre. Les attaques divines ne sont efficaces qu'en fin de fight et peuvent être counter par leur nombre (notamment si l'adversaire possède une action des dieux aussi). Ainsi, si vous parvenez à survivre à ce combo, ou si vous forcez le paladin a utiliser des béliers /boucliers au début, il ne pourra plus rien faire pour vous achever. 
Ainsi, il est puissant, mais si vous avez de quoi encaisser, il ne pourra plus rien faire après les combo (pas d'attaque qui fait mal et plus de souffle...). 
Ah aussi. Il est vraiment à chier en PvE, pour la raison au-dessus. Je suis quasiment incapable de vaincre le golem, qui me counter entièrement. 

2) le rapport avec le mage et les autres classes
Le paladin est plutôt fort contre le mage, tant que je suis capable de le détruire avec l'ultime + 2 divines (ce qui explique pourquoi je perds contre doc 😄). C'est un bon counter. Le problème est que je fais 1k de dégats en 4 tours... par contre avant et après : rien. Tout simplement parce qu'il suffit que je fasse une bouclier et je n'ai pas assez de souffle pour le combo final. Et que je ne peux sortir le grand jeu qu'une fois arrivé à 900- Pvs. Dans ce cas, il suffit que le mage joue intelligemment et ai gardé de quoi faire 2-3 sombres... avec un ou deux crit ils peuvent me terminer. En tous cas, une ultime qui crit faire du 400 de dégat par tour. 250 sans crit. La sombre qui crit + le poison est aussi à 400.... 
Bon. Le paladin est un counter du mage (et accessoirement, du canonnier). Mais... c'est tout. Il n'y a aucune classe que je peux fight. Vétéran et fantassin ont peut-être de quoi le surprendre mais il n'y en a aucun. Et ils ont une chance ! 
Un exemple (svp pas de blabla tu prends exemple sur un seul fight c'est pas la réalité... lisez) : 
cyrpa est vétéran. Je suis paladin, une classe qui claque en PvP. J'ai littéralement 10 niveaux de plus de lui, je commence avec 600 pvs de plus que lui. Je devrais l'exploser. Il fait 2 crit d'énergique / 3, c'est vraiment pas mal, mais pas si rare. 
Au final, n'ayant raté qu'une seule simple, ayant utilisé mon ultime et mes deux divines, il me restait 30 pvs à la fin du combat. 
Si quelqu'un avec 10 lvls de moins peut faire ça sans énorme coup de chance, alors un vétéran de mon niveau devrait pouvoir s'en sortir ! (mieux que le mage). Par exemple, qu'il fasse des chargées au début pour limiter les dégats, qu'il détruise un peu le paladin mais économise son souffle au maximum. Et ensuite, lors du combo du paladin, il suffit de faire le plus n'énergiques possibles et on résiste ! Après ce combo, vous roulez sur ce qui reste du paladin. C'est une stratégie intéressante ! (cyrpa essaie-là à notre prochain combat, tu es mon seul vétéran + ou - de mon niveau :)
Voilà, le problème est que le vétéran, et le fantassin se font détruire par le mage omniprésent dans le top gloire. 

Ah oui aussi. 
Si le paladin est si cheat que ça... pourquoi je suis le seul lvl 100+ du top gloire à avoir pris cette classe... à part Cyrpa, tous sont mages. Alors arrêtez de râler que la suprématie du mage soit remise en question par le cailloux dans votre botte que je suis en paladin 😂 
Pourquoi personne ne prend paladin ? C'est nul à chier en PvE 😉

3) le dilemme de l'attaque bélier
Mmm cette attaque avec le spam de l'étourdissement... parfois un gros problème pour certains joueurs lorsque la chance est contre vous. Après, cette attaque est la plus aléatoire du jeu. Elle peut stun l'adversaire ou l'assaillant, faire des dégats à l'adversaire... ou pas et à l'assaillant. C'est un gros meli melo de bazar et de chance... c'est marrant et n'est pas toujours favorable au paladin. C'est la seule attaque du jeu (avec la sabotage mais son récent buff la sort maintenant de cette catégorie) à pouvoir faire plus de mal à l'assaillant qu'à l'adversaire !
Personnellement je ne l'utilise pas. Car je combats des mages qui me volent mon souffle donc je n'ai pas vraiment le loisir de le dépenser à ça :)

Bref. En résumé : 
Le paladin est fort en PvP. Notamment pour la meilleure classe de la meta qu'est le mage, omniprésent. Il est un des seuls à pouvoir rivaliser avec lui, et en moyenne, le surpasser. Et tant mieux ! Cela permettra aux joueurs de pourquoi pas les faire se pencher sur le paladin, et une classe counter sera peut-être découverte ce qui régulera un peu la meta.
Mais... c'est vraiment nul en PvE.

**Reply by Electek** [reply]
je pense qu'il parle sourtout en t4

**Reply by Gabrielre14** [reply]
Bien évidemment. Je ne pourrais pas relater des faits concernant les lvl 80- sachant qu'il n'y a aucun mage et que je ne le vis pas. Etant lvl 116 et presque constamment dans le top 5 du top gloire, je donne mon avis sur ce qu'il se passe à haut niveau ^^ 
Mais je pense qu'à bas niveau le paladin fait des ravages... mais peut se faire counter plus facilement car les classes sont généralement plus fortes à bas niveau, lorsque les stats des joueurs sont encore assez déséquilibrées. Le mage est la seule classe plus puissante à haut niveau (attaque empoisonnée ne dépend pas des stats, attaque sombre plus forte lorsque l'attaque adverse) élevée...)

**Comment by Gabrielre14** [comment]
En vrai mon dernier paragraphe pas vraiment. 
Le paladin s'auto-counter à fond alors si je peux rester seul paladin du haut niveau le plus longtemps possible ça m'arrangerai 😁

**Comment by BananePlantain** [comment]
![image](https://github.com/DraftBot-A-Discord-Adventure/DraftBot/assets/126412670/2b060115-4bc5-4feb-a059-7dc236651169)
😴
